packages/ag-charts-enterprise/src/features/annotations/annotations.ts

```diff
@@ -476,6 +485,34 @@ export class Annotations extends _ModuleSupport.BaseModuleInstance implements _M
         );
     }
 
+    private createAnnotationDatumCopy({
+        node,
+        datum,
+    }: {
+        node: AnnotationScene;
+        datum: AnnotationProperties;
+    }): AnnotationProperties | undefined {
+        const { type } = datum;
+        if (type in annotationConfigs) {
+            const config = annotationConfigs[type];
+
+            const newDatum = new config.datum();
+            newDatum.set(datum.toJson());
+
+            const context = this.getAnnotationContext();
+            if (!context) {
+                return;
+            }
+
+            const offset = { x: -30, y: -30 }; // just above and to the left
```

Comment:
Since we're not using this offset on a bunch of these scenes, it might be better to not pass it down at all and instead calculate it inside each scene.

For some it will may just be the `-30,-30`, but I can see us wanting different numbers for different text annotation types and also different point scenes that have different sized shapes.